.github/workflows/ci.yml

```diff
@@ -62,6 +64,17 @@ jobs:
         run: yarn install --non-interactive --check-files --frozen-lockfile
       - name: Build library
         run: ./node_modules/.bin/gulp build
+      - name: Install github changelog generator
+        run: sudo gem install github_changelog_generator
+      - name: Export github token
+        run: export CHANGELOG_GITHUB_TOKEN=${{secrets.GITHUB_TOKEN}}
```

Comment:
I think this variable is not used anywhere, is it (also on the release task)?

Reply:
It is used by the github_changelog_generator in the gulp task.

Reply:
Ah, OK, thanks